internal/weapons/bow/hunterspath/hunterspath.go

```diff
@@ -0,0 +1,90 @@
+package hunterspath
+
+import (
+	"fmt"
+
+	"github.com/genshinsim/gcsim/pkg/core"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/event"
+	"github.com/genshinsim/gcsim/pkg/core/glog"
+	"github.com/genshinsim/gcsim/pkg/core/keys"
+	"github.com/genshinsim/gcsim/pkg/core/player/character"
+	"github.com/genshinsim/gcsim/pkg/core/player/weapon"
+	"github.com/genshinsim/gcsim/pkg/modifier"
+)
+
+const (
+	buffKey = "hunterspath-tireless-hunt"
+	icdKey = "hunterspath-icd"
+)
+
+func init() {
+	core.RegisterWeaponFunc(keys.HuntersPath, NewWeapon)
+}
+
+type Weapon struct {
+	Index int
+}
+
+func (w *Weapon) SetIndex(idx int) { w.Index = idx }
+func (w *Weapon) Init() error      { return nil }
+
+func NewWeapon(c *core.Core, char *character.CharWrapper, p weapon.WeaponProfile) (weapon.Weapon, error) {
+	// Gain 12% All Elemental DMG Bonus. Obtain the Tireless Hunt effect after
+	// hitting an opponent with a Charged Attack. This effect increases Charged
+	// Attack DMG by 160% of Elemental Mastery. This effect will be removed after
+	// 12 Charged Attacks or 10s. Only 1 instance of Tireless Hunt can be gained
+	// every 12s.
+	w := &Weapon{}
+	r := p.Refine
+
+	dmgBonus := 0.09 + 0.03*float64(r)
+	val := make([]float64, attributes.EndStatType)
+	for i := attributes.PyroP; i <= attributes.DendroP; i++ {
+		val[i] = dmgBonus
+	}
+	val[attributes.PhyP] = 0
```

Comment:
My fault on this one but the better fix would probably be to move Dendro index up to before Phys here: https://github.com/genshinsim/gcsim/blob/daa7e3c700413706a365a8f4e8da48d084da17ce/pkg/core/attributes/stats.go#L30

In theory shouldn't break anything since nothing should be hard coded? Maybe except UI? 🤔